### DISCUSSION TITLE
Scrypted Update 0.50.0 does not work from GUI

### DISCUSSION THREAD

**Discussion by skynan22** [tteck/Proxmox#1785] [Help Needed]
I have installed Scrypted LXC via `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/scrypted.sh)"`
 - everything works fine so far (currently installed version 0.41.0). Now there is an update available to 0.50.0 but after clicking the button "Update and Restart Scrypted", just a restart (without update) takes place, no update.

How can I update to version 0.50.0 from GUI or what else is to do, to get the new Scrypted Server version?
Thanks in advance!

**Comment by tteck** [comment]
I have just completed a fresh installation, and version 0.50.0 has been installed.
Following a snapshot or full backup, you can attempt the following commands:

```
systemctl stop scrypted
sudo -u root npx -y scrypted@latest install-server
systemctl start scrypted
```

**Reply by tteck** [reply]
Perhaps you should consider clearing your browser cache first?

**Reply by skynan22** [reply]
I tried using another Browser (Safari, FF) instead of Chrome but this does not make any difference - just restart, no update.
The same when I use mobile devices that were not used to open Scrypted management console.

**Reply by skynan22** [reply]
When I execute the commands above, within pve console, the following error occured:

<img width="1055" alt="Bildschirmfoto 2023-09-11 um 11 42 50" src="https://github.com/tteck/Proxmox/assets/6624641/3dec437c-8b9c-465e-ba46-4d24a0e3cc3a">

Version 0.41.0 still there - no update took place.

**Reply by tteck** [reply]
Please review the log for additional information; it appears that the issue may be related to `ffmpeg-static`. Is that something you added?

**Reply by tteck** [reply]
On a clean installation, the mentioned commands do not produce any errors.

**Reply by skynan22** [reply]
I added ffmpeg plugin, for testing reasons, correct.
After deleting the plugin it is still the same - no update.
The log file was renamed obviously, the latest I found says the following:

<img width="825" alt="Bildschirmfoto 2023-09-11 um 11 58 48" src="https://github.com/tteck/Proxmox/assets/6624641/f548fa3e-c36e-42b7-a2e2-a1063165cb4d">



What ist the best way to backup all settings of all the plugins installed and get a "clean" installation with version 0.50.0?

**Reply by tteck** [reply]
When you mention "deleted the plugin," are you referring to uninstalling it?
```
npm uninstall ffmpeg-static
```

**Reply by skynan22** [reply]
Correct

**Reply by tteck** [reply]
I am not well-versed in the operational details of Scrypted, so I will not be able to offer assistance on best backup practices.

**Reply by skynan22** [reply]
After uninstall ffmpeg-static error message still the same:

<img width="1001" alt="Bildschirmfoto 2023-09-11 um 12 19 19" src="https://github.com/tteck/Proxmox/assets/6624641/7b012688-ac49-4e0e-ae7f-a7f8aacb6225">

**Reply by tteck** [reply]
Did you install it globally?
```
npm cache clean --force
npm uninstall -g ffmpeg-static
```

**Reply by tteck** [reply]
If the above commands don't remove the path, and you have a backup, try:
```
rm -rf /root/.scrypted/node_modules/ffmpeg-static
```

**Reply by tteck** [reply]
> I added ffmpeg plugin

Is that something (plugins) that should be uninstalled within Scrypted?

**Reply by skynan22** [reply]
I ran npm commands above, no error occured.
I deleted the ffmpeg directory, with no error.

After that I tried to install new version the following error occured:

<img width="1008" alt="Bildschirmfoto 2023-09-11 um 13 51 37" src="https://github.com/tteck/Proxmox/assets/6624641/5c5da06a-27fe-46e6-92b7-bc98941e506b">

The directory /root/.scrypted/node_modules/ffmpeg-static was not there when I started the installation, the deletion was successfully completed.
